src/components/BuildLogPage/BuildLogPage.tsx

```diff
@@ -25,6 +26,9 @@ export const BuildLogPage = () => {
   const { buildId } = useParamsRequired();
 
   const { serviceContainerBuild } = useServiceContainerBuild();
+  const isBuilding = useMemo(() => serviceContainerBuild.data?.status === 'BUILDING', [serviceContainerBuild.data?.status]);
+
+  console.log(serviceContainerBuild);
```

Comment:
Do we want console log here or was it just for development?

Reply:
Just a leftover.